src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = days * 40;
```

Comment:
The 'BASE_PRICE' is calculated before the checks. It should be calculated after validating the number of days.